Services/Clustering/ClusteringConfig.cs

```diff
@@ -36,14 +39,20 @@ public class ClusteringConfig : IClusteringConfig
         private const int MIN_MASTER_LOCK_DURATION_MSECS = 10000;
         private const int MAX_MASTER_LOCK_DURATION_MSECS = 300000;
 
+        private const int DEFAULT_MAX_DEVICES_PER_NODE = 20000;
+
+        private const int MIN_MAX_DEVICES_PER_NODE = 1;
+
         private const int DEFAULT_PARTITION_SIZE = 1000;
         private const int MIN_MAX_PARTITION_SIZE = 1;
         private const int MAX_MAX_PARTITION_SIZE = 10000;
+        private const int MAX_MAX_DEVICES_PER_NODE = 1000000;
```

Comment:
please keep the constants grouped by scope, like the others. It helps with code readability

Reply:
All of my POC changes are in, so it looks like this can be fixed now.